os/include/tinyara/error_report_internal.h

```diff
@@ -0,0 +1,76 @@
+/****************************************************************************
+ *
+ * Copyright 2018 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#ifndef __INCLUDE_TINYARA_ERROR_REPORT_INTERNAL_H
+#define __INCLUDE_TINYARA_ERROR_REPORT_INTERNAL_H
+
+/* This file will be used to provide definitions to support
+ * error report framework
+ */
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <tinyara/config.h>
+#include <tinyara/sched.h>
+#include <sched.h>
```

Comment:
Why do you include both sched.h?

Reply:
Actually, we do not need sched.h here at all. Thanks for spotting this.